src/Components/ProductCard/ProductCard.jsx

```diff
@@ -64,18 +63,20 @@ export const ProductCard = ({ product }) => {
         </p>
       </div>
       <button
-        className="btn card-btn"
+        className="btn "
         onClick={() =>
           dispatch({
-            type: "addToCart",
-            payload: "ADD_TO_CART",
-            newItem: { ...product, quantity: 1 }
+            type: "ADD_TO_CART",
+            payload: { newItem: { ...product, quantity: 1 } },
           })
         }
       >
         add to Cart
       </button>
-      <button onClick={() => handleProductClick(product.id)} className="btn ">
+      <button

```

Comment:
I'd suggest to remove the _See Details_ button and instead make the card image clickable. Since it's a primary function, user don't have to find the button and click on it. They can simply click on card itself and see the details of the product. Just like any other E-commerce site.